ga4gh/frontend.py

```diff
@@ -261,6 +300,26 @@ def assertCorrectVersion(version):
         raise exceptions.VersionNotSupportedException()
 
 
+def authenticated(handler):
+    @functools.wraps(handler)
+    def checkAuthentication(*args, **kwargs):
+        """
+        The request should come with an Authorization header of form:
+        Authorization: Bearer <token>
+        We are interested in the token.
+
+        If there is no oidcClient, we are running naked and we don't check.
+        """
+        if app.oidcClient:
```

Comment:
This is checking if 'app.oidcClient is not None`, right? I would _much_ rather if these checks for `None` were done explicitly: see previous comment for reasoning. Also, I think the same thing is done below when setting `requestToken`.